File: Baekjoon/BaaarkingDog/0x03_array/10808.cpp
#include <iostream>
#include <string>

int main() {
  std::string S;
  std::cin >> S;
  int alphabet_array[26] = {0};

  for (char ch : S) {
    alphabet_array[ch - 'a']++;
  }

  for (int alphabet_count : alphabet_array) {
    std::cout << alphabet_count << " ";
  }

  return 0;
}

File: Baekjoon/BaaarkingDog/0x03_array/10808.py
# 10808: 알파벳 개수

alphabet_array = [0 for _ in range(26)] 

S = input()

for alphabet in S:
  alphabet_array[ord(alphabet) - ord('a')] += 1

for alphabet_count in alphabet_array:
  print(alphabet_count, end=" ")

